system/Helpers/url_helper.php

```diff
@@ -634,3 +634,37 @@ function mb_url_title(string $str, string $separator = '-', bool $lowercase = fa
 }
 
 //--------------------------------------------------------------------
+
+if (! function_exists('url_to'))
+{
+	/**
+	 * Get the full, absolute URL to a controller method
+	 * (with additional arguments)
+	 *
+	 * @param string $controller
+	 * @param mixed  ...$args
+	 *
+	 * @throws \CodeIgniter\Router\Exceptions\RouterException
+	 *
+	 * @return string
+	 */
+	function url_to(string $controller, ...$args): string
+	{
+		if (! $route = route_to($controller, ...$args))
+		{
+			throw new \CodeIgniter\Router\Exceptions\RouterException(lang('HTTP.invalidRoute', [$controller]));
```

Comment:
`HTTP.invalidRoute` seems for redirect response, probably use `HTTP.controllerNotFound` with uses explode `::` from controller and method variable definition.

Reply:
Will change that, thank you